### ISSUE TITLE
Doesn't extract keys from Interpolation component with react-i18next

### ISSUE THREAD

**Issue by isld** [i18next/i18next-parser#57]
Can't be much more descriptive than in the title. Here is a code snippet:

```
<div className="flex-item-m-2 recent-activity-item-footer">
  <Interpolate i18nKey='Total {{amount}}' useDangerouslySetInnerHTML={true} amount={totalDocs} />
</div>
```

Sadly, the string 'Total {{amount}}' never makes it to the JSON output of the parser.

**Comment by karellm** [comment]
@isld I looked into this. It wouldn't be too hard to add but I'm somewhat reluctant to add more trackers to the code. It might be time to have a better configuration to support different needs (people using react likely don't care about html parsing or handlebars). Would you be interested in submitting a PR?

**Comment by hotdang-ca** [comment]
I may take this on as well...

**Comment by davelandry** [comment]
I was able to override the default regex to grab both <Interpolate> components and translate functions. It's rudimentary, and assumes the codebase enforces double-quotations, but it should be a start:
```sh
-p "i18nKey=\"([^\"]+)\"|t\(\"([^\"]+)\""
```

**Comment by tkrotoff** [comment]
The problem is more complex, you have multiple cases in order to support react-i18next:
```JSX
<Trans i18nKey="hello">Hello <strong>World!</strong></Trans>
<Trans>Hello, <strong>World!</strong></Trans>
```

```JSON
{
  "hello": "Bonjour <1>le monde !</1>",
  "Hello, <1>World!</1>": "Bonjour <1>le monde !</1>"
}
```

**Comment by karellm** [comment]
I will write a React Lexer for the `1.x` release. I'm currently working on matching existing features and should land a beta this week.

**Comment by karellm** [comment]
I just published `1.0.0-beta2` which adds support for react. I invite you to test it and open a new ticket if there are issues.
